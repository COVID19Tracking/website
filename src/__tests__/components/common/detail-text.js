import React from 'react'
import renderer from 'react-test-renderer'
import DetailText from '../../../components/common/detail-text'

describe('Components : Common: Detail text', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <DetailText>
          <p>Sample text</p>
        </DetailText>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
