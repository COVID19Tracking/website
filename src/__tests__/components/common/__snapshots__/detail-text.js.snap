// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Detail text renders correctly 1`] = `
<div
  className="detail-text detailText"
>
  <p>
    Sample text
  </p>
</div>
`;
